GW2EIEvtcParser/EncounterLogic/Raids/W7/PeerlessQadim.cs

```diff
@@ -50,26 +51,56 @@ public PeerlessQadim(int triggerID) : base(triggerID)
             EncounterID |= 0x000003;
         }
 
+        protected override List<int> GetTargetsIDs()
+        {
+            return new List<int>()
+            {
+                (int)ArcDPSEnums.TargetID.PeerlessQadim,
+                (int)ArcDPSEnums.TrashID.FriendlyPeerlessQadimPylon,
```

Comment:
A friendly as target? Wouldn't it make more sense to put the other one?

Reply:
You cc the friendly pylons, the hostile ones can't be hit and only deal the pulsing aura damage, example:

![6151](https://github.com/baaron4/GW2-Elite-Insights-Parser/assets/23245218/d712a580-e77f-474a-aa22-4da0fdcc1397)

North gets 3 breakbar phases (100%, 80%, 60%), south west 4 (40%) south east 5 (30%)

Reply:
In that case we should rename them as that means the "friendlyPylons" are not always friendly.
Something like PeerlessQadimPylon for the "friendly" and PeerlessQadimAuraPylon for the other.

Reply:
![FriendlyPylon](https://github.com/baaron4/GW2-Elite-Insights-Parser/assets/23245218/a87a4bb9-27f3-4b70-a2b3-40f77872c5b0)
![HostilePylon](https://github.com/baaron4/GW2-Elite-Insights-Parser/assets/23245218/3ef9f4df-ba43-4ee1-b9f3-19fad1373e8e)

I've updated their icons to these 2, should i just use this instead then?

![gw751](https://github.com/baaron4/GW2-Elite-Insights-Parser/assets/23245218/7bde90ed-f1c3-4485-abc3-09ea42b456d3)